google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -912,22 +915,24 @@ public void testListBlobsEmptySelectedFields() throws InterruptedException {
 
   @Test(timeout = 7500)
   public void testListBlobRequesterPays() throws InterruptedException {
+    unsetRequesterPays();
     BlobInfo blob1 =
-        BlobInfo.newBuilder(BUCKET, "test-list-blobs-empty-selected-fields-blob1")
+        BlobInfo.newBuilder(BUCKET_REQUESTER_PAYS, "test-list-blobs-empty-selected-fields-blob1")
             .setContentType(CONTENT_TYPE)
             .build();
     assertNotNull(storage.create(blob1));
 
     // Test listing a Requester Pays bucket.
     Bucket remoteBucket =
-        storage.get(BUCKET, Storage.BucketGetOption.fields(BucketField.ID, BucketField.BILLING));
-    assertFalse(remoteBucket.requesterPays());
+        storage.get(BUCKET_REQUESTER_PAYS, Storage.BucketGetOption.fields(BucketField.ID, BucketField.BILLING));
+
+    assertTrue(remoteBucket.requesterPays() == null || !remoteBucket.requesterPays());
```

Comment:
use && instead of ||

Reply:
Why? What is the benefit of ` asserFalse(requestPays() != null && requestPays())`?    For me it's hard to understand

Reply:
Beacuse because as you know `remoteBucket.requesterPays() == null` and `!remoteBucket.requesterPays()` both condition return `true `i guess correct? so its better to check something like  `assertTrue(remoteBucket.requesterPays() == null &&  !remoteBucket.requesterPays());`

Reply:
yes correct but you should know about that case and add checks for that.

Reply:
`assertTrue(remoteBucket.requesterPays() == null || !remoteBucket.requesterPays())` verifies that `remoteBucket.requesterPays()` returns either `null` or `False`. That's all.